lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -829,17 +899,29 @@ def remove_toolitem(self, name):
                     self._groups[group].remove(toolitem)
         del self._toolitems[name]
 
+    def set_flow(self, flow):
```

Comment:
@OceanWolf this is not in `ContainerBase`, and should be if it's a public method.